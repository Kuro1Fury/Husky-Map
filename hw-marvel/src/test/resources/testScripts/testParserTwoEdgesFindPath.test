# Tests that the path between two edges in the graph constructed by parser can be found

LoadGraph chars threeNodesTwoEdges.csv

FindPath chars Luigi Link

FindPath chars Link Luigi