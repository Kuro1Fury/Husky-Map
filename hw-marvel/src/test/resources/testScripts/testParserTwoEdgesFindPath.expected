# Tests that the path between two edges in the graph constructed by parser can be found

loaded graph chars

path from Luigi to Link:
Luigi to Mario via Super-Mario-Bros
Mario to Link via Super-Smash-Bros

path from Link to Luigi:
Link to Mario via Super-Smash-Bros
Mario to Luigi via Super-Mario-Bros
